src/utils/itemValidator.js

```diff
@@ -0,0 +1,27 @@
+export const isInputEmpty = (userInput) => {
+	const trimmedUserInput = userInput.trim();
+	return !trimmedUserInput;
+};
+
+export const removePunctuation = (inputItem) => {
+	const punctuation = /[.,?!@#$%^*]/g;
+	const itemWithoutPunctuation = inputItem.replace(punctuation, '');
+	const processedItem = itemWithoutPunctuation
+		.split(' ')
+		.join('')
+		.toLowerCase();
+
+	return processedItem;
+};
```

Comment:
We can make this more concise by including `\s` in the regex which will also check for any whitespaces. That way there's no need to do extra work by chaining `.split()` and `.join()`.

```javascript
export const removePunctuation = (inputItem) => {
  const punctuationAndSpaces = /[.,?!@#$%^*\s]/g;
  const processedItem = inputItem.replace(punctuationAndSpaces, '').toLowerCase();

  return processedItem;
};
```

Reply:
Thats a good refactor, I'll implement this.

Reply:
Hey Jose! Thanks for the suggestion, I have implemented your refactor.